ocrd_tesserocr/common.py

```diff
@@ -254,6 +254,10 @@ def make_join(polygons, scale=20):
         bridgep = LineString(nearest).buffer(max(1, scale/5), resolution=1)
         polygons.append(bridgep)
     jointp = unary_union(polygons)
+    
+    if isinstance(jointp, MultiPolygon):
+        jointp = max(jointp.geoms, key=lambda p: p.area)  # Choose the largest polygon
```

Comment:
One simple way to share example polygon data is by using Shapely's `wkt` serialization, which can easily be printed, plotted, re-imported etc. So here, e.g. if you do...
```suggestion
        for geom in jointp.geoms:
            print(geom.wkt)
        jointp = max(jointp.geoms, key=lambda p: p.area)  # Choose the largest polygon
```
...and then send me the stdout, that would really help.